docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
+
+  Creates a :ref:`password reset request <data-password-resets>` for the user
+  with the email address provided in the request body.
```

Comment:
We should probably mention that this will also email the token to the email address if the user exists.